include/fizzy/fizzy.h

```diff
@@ -125,6 +125,12 @@ FizzyInstance* fizzy_instantiate(const FizzyModule* module,
 /// If passed pointer is NULL, has no effect.
 void fizzy_free_instance(FizzyInstance* instance);
 
+/// Get pointer to module of an instance.
+///
+/// @note The returned pointer respresents non-owning, "view"-access to the module and must not be
+/// passed to fizzy_free_module.
+const FizzyModule* fizzy_get_instance_module(FizzyInstance* instance);
```

Comment:
I think this may be better wording, but we already have `get_instance_memory`, so I suggest we go with what is the PR and then rename all of them at once if we want.